whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapServiceAclTestIntegration.java

```diff
@@ -120,4 +170,167 @@ public void lookup_mntner_acl_not_counted() throws Exception {
         assertThat(((ArrayList)entity.getVCardArray().get(1)).get(2).toString(), is("[kind, {}, text, individual]"));
         assertThat(testPersonalObjectAccounting.getQueriedPersonalObjects(InetAddress.getByName(LOCALHOST)), is(0));
     }
+
+    @Test
+    public void lookup_inetnum_filtered_emails_acl_not_counted() throws Exception {
```

Comment:
Can you also test that related entity e-mail is redacted?

Reply:
Yes, I did it